src/client/datascience/jupyter/kernels/kernel.ts

```diff
@@ -364,10 +364,6 @@ export class Kernel implements IKernel {
             traceInfoIf(isCI, 'Step I');
             await this.disableJedi();
             traceInfoIf(isCI, 'Step J');
-            if (this.resourceUri) {
-                await this.notebook.setLaunchingFile(this.resourceUri.fsPath);
```

Comment:
In lin 361, we're already setting the working directory (code is identical), hence no need to call notebook API again.

Reply:
Except on line 361, the resourceUri.fsPath isn't passed, so the logic will be different.

Reply:
Good one, fixed.